src/services/messengerSync/producer.js

```diff
@@ -0,0 +1,80 @@
+const { Configuration } = require('@schul-cloud/commons');
+const { createChannel } = require('../../utils/rabbitmq');
+
+const QUEUE_INTERNAL = Configuration.get('RABBITMQ_MATRIX_QUEUE_INTERNAL');
+
+let app;
+let channel;
+
+const sendToQueue = (message) => {
+	const msgJson = JSON.stringify(message);
+	const msgBuffer = Buffer.from(msgJson);
+	channel.sendToQueue(QUEUE_INTERNAL, msgBuffer, { persistent: true });
+};
+
+const requestFullSchoolSync = (school) => {
+	const message = {
+		action: 'syncSchool',
+		schoolId: school._id,
+		fullSync: true,
+	};
+	sendToQueue(message);
+};
+
+const requestSyncForEachCourseUser = async (course) => {
```

Comment:
however, there is a second parameter context, which is not used, but highlights this and other methods below to be an feathers event, add `_context` as second param https://docs.feathersjs.com/api/events.html eslint should accept this